### PR TITLE
Add --disable-unicode flag

### PR DESCRIPTION
**Pull request by margaretgorguissian**
This PR addresses issue #2.

Add a `--disable-unicode` flag, and only fuzz with ASCII characters when the flag is included. This PR adds an addition `unicode_enabled` setting to `Settings()`, and its default is true.

### CODE REVIEW

File: fuzz_lightyear/settings.py [hunk 1]
```diff
@@ -7,6 +7,15 @@
...
         self.seed = random.getrandbits(128)    # type: int
+        self.unicode_enabled = True            # type: bool
+
+    @property
+    def unicode_enabled(self) -> bool:
+        return self._unicode_enabled
+
+    @unicode_enabled.setter
+    def unicode_enabled(self, value: bool) -> None:
+        self._unicode_enabled = value
```
Comment (domanchi) [inline fuzz_lightyear/settings.py]: Is there a specific reason why we can't just use the raw value? It doesn't look like the getter or setter is doing anything additional....

File: fuzz_lightyear/fuzzer.py [hunk 2]
```diff
@@ -102,10 +103,14 @@ def _fuzz_string(
 
     # TODO: Handle a bunch of swagger string formats.
     # https://swagger.io/docs/specification/data-models/data-types/#string
+    kwargs = {}                                     # type: Dict[str, Any]
+
     if parameter.get('required', required):
-        return st.text(min_size=1)
+        kwargs['min_size'] = 1
+    if not get_settings().unicode_enabled:
+        kwargs['alphabet'] = string.ascii_letters
```
Comment (domanchi) [inline fuzz_lightyear/fuzzer.py]: Do we want to include numbers too? Or punctuation? Whitespace?

Comment (margaretgorguissian) [reply fuzz_lightyear/fuzzer.py]: D'oh. I'll change to `string.printable`.

File: fuzz_lightyear/settings.py (hunk 1 above)
Comment (margaretgorguissian) [reply fuzz_lightyear/settings.py]: 👍 I guess there's no reason, only to keep it consistent with `seed`. I'll change.

**Pull request merged**